### DISCUSSION TITLE
FORMKIT FILE VALIDATION

### DISCUSSION THREAD

**Discussion by yogendrarr** [formkit/formkit#1378] [General]
when i select file and then upload file from api. then when i come back to same page then uploaded document fetch but , ISSUE is that -> it should not validate if document already uploaded so how to do in file type of formkit.

`  <FormKit
      type="file"
      label="Drivers license"
      name="license"
      help="Please add a scan of your driver’s license"
      accept=".jpg,.png,.pdf"
      validation="required"
    />`

**Comment by fenilli** [comment]
If you want to remove validation you can just do something like `:validation="fileExists ? '' : 'required'"`
